.husky/pre-push

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env sh
```

Comment:
The pre-commit hook should be enough, we prob don't need this since anything pushed will already have been linted with pre-commit.

Reply:
> The pre-commit hook should be enough, we prob don't need this since anything pushed will already have been linted with pre-commit

Agreed. I have taken out the `pre-push` hook in https://github.com/cypress-io/cypress-example-kitchensink/pull/626/commits/9286a3d5ef08eaa3e05e6af0260d825661aa14a4.